client/core/core.go

```diff
@@ -3215,6 +3273,34 @@ func (c *Core) initialize() {
 	c.refreshUser()
 }
 
+func (c *Core) verifyAccount(acct *db.AccountInfo) bool {
+	host, err := addrHost(acct.Host)
+	if err != nil {
+		c.log.Errorf("skipping loading of %s due to address parse error: %v", acct.Host, err)
+		return false
+	}
+	dc, err := c.connectDEX(acct)
+	if err != nil {
+		c.log.Errorf("error connecting to DEX %s: %v", acct.Host, err)
+		return false
+	}
+	c.log.Debugf("connectDEX for %s completed, checking account...", acct.Host)
+	if !acct.Paid {
+		if len(acct.FeeCoin) == 0 {
+			// Register should have set this when creating the account
```

Comment:
If not paid, but there is a fee coin, we'd need to do something like `checkUnpaidFees` or `reFee` (not sure), otherwise dexc would need a restart to finish this up.

Reply:
Ah, I see you've named that as next step in your comment. https://github.com/decred/dcrdex/pull/885#issuecomment-749339222 Sounds good.